src/main/java/lt/andriaus/hangman/domain/GameException.java

```diff
@@ -1,21 +1,17 @@
 package lt.andriaus.hangman.domain;
 
-class GameException extends RuntimeException {
-    enum Exceptions {
-        GameIsAlreadyOverException,
-        SymbolIsNotAlphabeticException
-    }
+public class GameException extends RuntimeException {
 
     private GameException(String message) {
         super(message);
     }
 
     public static GameException GameIsAlreadyOverException() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.